SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -30,6 +39,18 @@ DECLARE_CYCLE_STAT(TEXT("DrawText"), STAT_DrawText, STATGROUP_SpatialDebugger);
 DECLARE_CYCLE_STAT(TEXT("BuildText"), STAT_BuildText, STATGROUP_SpatialDebugger);
 DECLARE_CYCLE_STAT(TEXT("SortingActors"), STAT_SortingActors, STATGROUP_SpatialDebugger);
 
+USTRUCT()
+struct FWorkerRegionInfo
+{
+	GENERATED_BODY()
+
+	UPROPERTY()
```

Comment:
are these UPROPERTY tags required?

Reply:
I believe so, this is so that they are replicated with the USTRUCT